aicspylibczi/tests/test_czi_file.py

```diff
@@ -51,14 +51,16 @@ def test_metadata(data_dir, fname, xp_query, expected):
 
 
 @pytest.mark.parametrize("fname, expected", [
-    ('s_1_t_1_c_1_z_1.czi', (1, 1, 325, 475)),
-    ('s_3_t_1_c_3_z_5.czi', (1, 3, 3, 5, 325, 475)),
+    ('s_1_t_1_c_1_z_1.czi', ((1, 1, 325, 475), [('B', 1), ('C', 1), ('Y', 325), ('X', 475)])),
```

Comment:
I would split this into `expected_shape` and `expected_size` so that you don't have to do tuple expansion down below like `expected[0]`